### DISCUSSION TITLE
Historical Scorecard Data (prior to 2021)

### DISCUSSION THREAD

**Discussion by liaochris** [ossf/scorecard#4390] [Q&A] [answered]
If I want to get scorecard data for a github repository prior to 2021 (say, for example, for the commit SHA corresponding to March 2019), how would I go about doing that? I don't see an option for getting scorecard scores for specific instances of a git repo, only the current version. Thank you, and please let me know if I'm missing something!

**Comment by spencerschrock** [comment, marked as answer]
You can use the `--commit <SHA>` CLI option, but some checks will use current repo settings (such as Branch-Protection).
